account_move_line_report_xls/report/__init__.py

```diff
@@ -21,3 +21,4 @@
 ##############################################################################
 
 from . import move_line_list_xls
+
```

Comment:
Please don't add empty line at EOF

    ./account_move_line_report_xls/report/__init__.py:24:1: W391 blank line at end of file